apigee-test-shared/README.md

```diff
@@ -0,0 +1,9 @@
+# Apigee-test-shared
+Docker image used to run unit tests on Apigee proxies
+
```

Comment:
could you add documentation about the expected folder structure in the target repo? Specifically, there should be a 'tests' folder under 'src'